packages/zero-runtime/package.json

```diff
@@ -58,7 +58,8 @@
       "sx": "./exports/sx.js",
       "keyframes": "./exports/keyframes.js",
       "generateAtomics": "./exports/generateAtomics.js",
-      "css": "./exports/css.js"
+      "css": "./exports/css.js",
+      "createUseThemeProps": "./exports/useThemeProps.js"
```

Comment:
Let's keep the filename same as the function name. It should be createUseThemeProps.ts